.eslintrc

```diff
@@ -11,5 +11,14 @@
   },
   "settings": {
     "import/resolver": "meteor",
+  },
+  "globals": {
```

Comment:
There are a couple of other ways we could do this (@pauldowman do you have an opinion here?):

```
{
    "env": {
        "mocha": true
    }
}
```

The above is maybe a little cleaner/semantic. One side-effect of setting these is that it affects non-test code. E.g., lodash has `before` and `after` methods, and lodash methods may be imported by name directly. `app` also seems like it could turn up. If we want to avoid that, we could ask testers to set this at the top of each test file:

```
/* eslint-env mocha */
```

That might be my preferred method, but I'm interested what anyone else thinks.